assets/airports/kcvg.json

```diff
@@ -0,0 +1,256 @@
+		],
+		"delay": [10, 10],
+		"ils": [true, true]
+	}, {
+		"name": ["18L", "36R"],
+		"name_offset": [
+			[0, 0],
+			[0, 0]
+		],
+		"end": [
+			["N39d3.35", "W84d38.80", "886.2ft"],
+			["N39d1.70", "W84d38.81", "896.2ft"]
+		],
+		"delay": [10, 10],
+		"ils": [true, true]
+	}, {
+		"name": ["18R", "36L"],
+		"name_offset": [
+			[0, 0],
+			[0, 0]
+		],
+		"end": [
+			["N39d4.25", "W84d41.02", "865.4ft"],
+			["N39d2.94", "W84d41.03", "873.4ft"]
+		],
+		"delay": [10, 10],
+		"ils": [true, true]
+	}],
+	"sids": {
+		"CHCLL4": {
+			"icao": "CHCLL4",
+			"name": "Churchill Four",
+			"rwy": {
+				"18L": ["CAMUM", "ZEGUL"],
+				"18R": ["FANSA"],
+				"18C": [
+					["FLOEE", "A14+"], "FEBLU"
+				],
+				"27": [
+					["BONLE", "A14+"], "CULED"
+				],
+				"36L": ["HOBAL", "BOUMR", "CULED"],
+				"36C": ["HOBAL", "BOUMR", "CULED"],
+				"36R": [
+					["HAGSS", "A14+"], "EVAME", "CEMLA", "FRTZY", "CULED"
+				]
+			},
+			"body": [
+				["CHCLL", "A85+"],
+				["PONIE", "A104+"]
+			],
+			"exitPoints": {
+				"IIU": [
+					["IIU", "A210+"]
+				],
+				"BWG": [
+					["SKINN", "A210+"], "BWG"
+				]
+			},
+			"draw": [
+				["BONLE", "CULED"],
+				["HOBAL", "BOUMR", "CULED"],
+				["HAGSS", "EVAME", "CEMLA", "FRTZY", "CULED"],
+				["CULED", "CHCLL"],
+				["FANSA", "CHCLL"],
+				["CAMUM", "ZEGUL", "CHCLL"],
+				["FLOEE", "FEBLU", "CHCLL"],
+				["CHCLL", "PONIE", "IIU*", "SKINN", "BWG*"]
+			]
+		},
+		"BNGLE4": {
+			"icao": "BNGLE4",
+			"name": "Bangle Four",
+			"rwy": {
+				"18L": ["CAMUM", "ANTEO", "CODKO", "JBETH", "FOPAS"],
+				"18C": [
+					["FLOEE", "A14+"], "ZUGLU", "MIPIE", "HERMM", "AMMOT"
+				],
+				"18R": ["FANSA", "ZUGLU", "MIPIE", "HERMM", "AMMOT"],
+				"27": [
+					["BONLE", "A14+"], "AMMOT"
+				],
+				"36R": [
+					["HAGSS", "A14+"]
+				],
+				"36C": ["CIGMA", "DJOHN"],
+				"36L": ["CIGMA", "DJOHN"]
+			},
+			"body": [
+				["BNGLE", "A80+"]
+			],
+			"exitPoints": {
+				"RIKLE": [
+					["RIKLE", "A180+"]
+				],
+				"DJB": [
+					["RIKLE", "A180+"], "DJB"
+				]
+			},
+			"draw": [
+				["CAMUM", "ANTEO", "CODKO", "JBETH", "FOPAS", "BNGLE"],
+				["FLOEE", "ZUGLU"],
+				["FANSA", "ZUGLU"],
+				["ZUGLU", "MIPIE", "HERMM", "AMMOT"],
+				["BONLE", "AMMOT"],
+				["AMMOT", "BNGLE"],
+				["HAGSS", "BNGLE"],
+				["CIGMA", "DJOHN", "BNGLE"],
+				["BNGLE", "RIKLE*", "DJB*"]
+			]
+		}
+	},
+	"spawnPatterns": [{
+		"origin": "KCVG",
+		"destination": "",
+		"category": "departure",
+		"route": "KCVG.CHCLL.IIU",
+		"altitude": 0,
+		"speed": 0,
+		"method": "random",
+		"rate": 4,
+		"airlines": [
+			["dal", 51],
+			["aal", 17],
+			["ual", 11],
+			["aca", 2],
+			["swa", 1]
+		]
+	}, {
+		"origin": "KCVG",
+		"destination": "",
+		"category": "departure",
+		"route": "KCVG.CHCLL.BWG",
+		"altitude": 0,
+		"speed": 0,
+		"method": "random",
+		"rate": 4,
+		"airlines": [
+			["dal", 51],
+			["aal", 17],
+			["ual", 11],
+			["aca", 2],
+			["swa", 1]
+		]
+	}]
+}
```

Comment:
All of these new line problems were from JSONLint, copy/paste into JSONLint, copy/paste back into kcvg.json file. It did that automatically.